modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/ListEntries.es.js

```diff
@@ -43,14 +44,20 @@ export default function ListEntries() {
 
 	const permissions = usePermissions();
 
-	const formColumns = columns.map(({value, ...column}) => ({
-		...column,
-		value: getLocalizedUserPreferenceValue(
-			value,
-			userLanguageId,
-			dataDefinition.defaultLanguageId
-		),
-	}));
+	const formColumns = [
+		...columns.map(({value, ...column}) => ({
+			...column,
+			value: getLocalizedUserPreferenceValue(
+				value,
+				userLanguageId,
+				dataDefinition.defaultLanguageId
+			),
+		})),
+		{
+			key: 'status',
+			value: Liferay.Language.get('status'),
+		},
```

Comment:
@albinojunior, are we going to add the status column to the AppBuilder side as well? Or should we add a flag to hide this column?

Reply:
Yes, we are adding the status column to the App Builder side and it should work like at the Forms. All entries submitted on deployed app without a workflow trigged should display "Approved" on the status column by default.

But I can confirm it with the PM team and proceed with this PR.

Reply:
Thanks, Bino